### ISSUE TITLE
pdfpc does not open recognize links anymore 

### ISSUE THREAD

**Issue by diegit** [pdfpc/pdfpc#313]
I open videos in an external player (vlc in my case) via \href .This does not work anymore in newer versions.

### Expected Behaviour
Open videos in external player when clicking link

### Actual Behaviour
Link not clickable

### If it's a bug: Steps to reproduce the behaviour (e.g. a minimal working example for the PDF)

```
\documentclass[11pt,a4paper]{beamer}
\usepackage[utf8]{inputenc}
\usepackage{xstring}
\usepackage{hyperref}

\newcommand\myPth{file:///*AbsPath*/}
\begin{document}

\begin{frame}
\href{\myPth somevid.mp4}{Click }
\end{frame}
\end{document}
```
myPth is the absolute path of the video location, for example

` \newcommand\myPth{file:///home/user/presentation/example/}`

#### pdfpc version
working 4.0.7 and below
not working 4.0.8 and above

#### Used Distribution (GTK version, vala version)
Opensuse Tumbleweed (all versions where compiled on this system)

#### Display Server (X/Wayland)
X

#### HiDPI Setting (yes/no)
no

**Comment by AndreasBilke** [comment]
Does pr #314 helps?

**Comment by diegit** [comment]
thank you for the fast response!

It fixes it on the presenter screen, but not on the main screen. I often have notes in my presentations and then the clickable area is at the location, where the link would be without the notes.

**Comment by AndreasBilke** [comment]
What do you mean with main screen? The presentation on the beamer?

**Comment by diegit** [comment]
@AndreasBilke 
yes, I mean the presentation on the beamer. With the older versions, the links where also clickable from that screen.

I just tested it again and not all pdfs seem to work. I attached a pdf where the link is not clickable from the presenter screen, while it works with other programs (okular).
[test.pdf](https://github.com/pdfpc/pdfpc/files/1467729/test.pdf)

@fnevgeny 
There are two different reasons: the first one is that videos are duplicated on the presenter screen and I can't see my notes then ( https://github.com/pdfpc/pdfpc/issues/292 ). 
The second is that I sometimes want to show a fullscreen video while I'm on a slide. For example I have an equation and want to show how a parameter changes the outcome. Then I just have links on my slide with alpha = 1, alpha = 2 etc. As soon as I close a video the audience can see the equations again and I can add other terms or add more explanations.
